BritneyS/comic-web-detail
# Main view scaffolding

## What does this PR do :question:
This PR includes the scaffolding for the main view of the app, including the "Heroes" and "Villains" tabs. Sample `heroes` and `villains` objects were created to mimic what will be eventually be received from the API.

Also, the name of the app was changed to the "HeroDex".

## How to test it :microscope:
1. Check out this branch.
2. Run `npm install && npm start`.
3. Open the browser to http://localhost:3000/ if the page doesn't open automatically.
4. Click both the "Heroes" and "Villains" tabs. Four images representing either comic book heroes or villains should appear each time.
5. Click any image multiple times. The sidebar section should appear and disappear (see gif).
6. Resize the width of the window to its smallest size, then click any image again. The sidebar section should appear over the entire screen.
7. Click the `X` button in the top right corner of the screen. The sidebar should disappear again.



## Any background info you would like to include :white_check_mark:
N/A


## Screenshots (if applicable) :camera:
![comic-web-detail-pr-mainview-1](https://user-images.githubusercontent.com/8409475/60762244-0cc74500-a029-11e9-8b0a-328cbe77021e.gif)
![comic-web-detail-pr-mainview-2](https://user-images.githubusercontent.com/8409475/60762247-12bd2600-a029-11e9-98bc-ace96331eb69.gif)
